test/functional/bip68-112-113-p2p.py

```diff
@@ -152,24 +137,18 @@ def create_test_block(self, txs, version=536870912):
         return block
 
     def create_bip68txs(self, bip68inputs, txversion, locktime_delta=0):
-        txs = []
+        """Returns a dictionary of bip68 transactions with different bits set."""
```

Comment:
In commit "Remove nested loops from bip68-112-113-p2p.py"

It seems better to continue returning a list than a dictionary. A dictionary is just slower and introduces nondeterminism in the test for no reason.

Reply:
yes - returning a list is much better. Changed. Thanks